template/package.json

```diff
@@ -46,7 +41,11 @@
     "eslint-plugin-react": "latest",
     "eslint-plugin-react-hooks": "latest",
     "extra-watch-webpack-plugin": "^1.0.3",
-    "styled-jsx": "^5.1.2",
+    "i18next": "23.10.1",
```

Comment:
Same here, why not allowing minor/patch updates?
And we should be consistent with the rest of the dependencies if we end up fixing the versions

Reply:
If i18next adds a new function in 23.11.0 (semver minor), it would appear usable during development but fail when deployed to a jahia instance